show/main.py

```diff
@@ -2806,5 +2806,41 @@ def features():
         body.append([key, status_data[key]['status']])
     click.echo(tabulate(body, header))
 
+#
+# 'container' group
+#
+@cli.group(cls=AliasedGroup, default_if_no_args=False)
```

Comment:
There is no need for this group to be aliased. Try using this definition instead:

```
@cli.group(name='container', invoke_without_command=False)
```

Reply:
Fixed.